src/operator/controllers/kafkaserverconfig_controller.go

```diff
@@ -33,33 +36,43 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"time"
 )
 
 const (
 	finalizerName                              = "intents.otterize.com/kafkaserverconfig-finalizer"
 	ReasonIntentsOperatorIdentityResolveFailed = "IntentsOperatorIdentityResolveFailed"
 	ReasonApplyingKafkaServerConfigFailed      = "ApplyingKafkaServerConfigFailed"
 	ReasonAppliedKafkaServerConfigFailed       = "AppliedKafkaServerConfigFailed"
+	IntentsOperatorSource                      = "intents-operator"
 )
 
 // KafkaServerConfigReconciler reconciles a KafkaServerConfig object
 type KafkaServerConfigReconciler struct {
 	client.Client
 	Scheme               *runtime.Scheme
-	ServersStore         *kafkaacls.ServersStore
+	ServersStore         kafkaacls.ServersStore
 	operatorPodName      string
 	operatorPodNamespace string
+	otterizeClient       otterizecloud.CloudApi
 	injectablerecorder.InjectableRecorder
 }
 
-func NewKafkaServerConfigReconciler(client client.Client, scheme *runtime.Scheme, serversStore *kafkaacls.ServersStore,
-	operatorPodName string, operatorPodNameSpace string) *KafkaServerConfigReconciler {
+func NewKafkaServerConfigReconciler(
+	client client.Client,
+	scheme *runtime.Scheme,
+	serversStore kafkaacls.ServersStore,
+	operatorPodName string,
+	operatorPodNameSpace string,
+	otterizeCloudClient otterizecloud.CloudApi,
```

Comment:
Please name this either CloudClient (I think it more accurately reflects what it is, even as an interface) or CloudAPI (following the Golang style guide - abbreviations should be all uppercase, e.g. CloudAPI not CloudApi, GraphQL not GraphQl, and so on)